src/detail/serializer.hpp

```diff
@@ -1,17 +1,18 @@
 #ifndef NLOHMANN_JSON_DETAIL_SERIALIZER_HPP
 #define NLOHMANN_JSON_DETAIL_SERIALIZER_HPP
 
-#include <algorithm> // reverse, remove, fill, find, none_of
-#include <array> // array
-#include <cassert> // assert
-#include <clocale> // localeconv, lconv
-#include <cmath> // labs, isfinite, isnan, signbit
-#include <cstddef> // size_t, ptrdiff_t
-#include <cstdint> // uint8_t
-#include <cstdio> // snprintf
-#include <iterator> // next
-#include <limits> // numeric_limits
-#include <string> // string
+#include <algorithm>   // reverse, remove, fill, find, none_of
```

Comment:
It seems that `astyle` doesn't reindent those comments, I shall update that manually after rebasing on `develop`.